docs/basic-features/eslint.md

```diff
@@ -42,25 +42,36 @@ We recommend using an appropriate [integration](https://eslint.org/docs/user-gui
 
 Once ESLint has been set up, it will automatically run during every build (`next build`). Errors will fail the build, while warnings will not.
 
-If you do not want ESLint to run as a build step, it can be disabled using the `--no-lint` flag:
-
-```bash
-next build --no-lint
-```
-
-**This is not recommended** unless you have configured ESLint to run in a separate part of your workflow (for example, in CI or a pre-commit hook).
+If you do not want ESLint to run as a build step, refer to the documentation for [Ignoring ESLint](/docs/api-reference/next.config.js/ignoring-eslint.md):
 
 ## Linting Custom Directories
 
-By default, Next.js will only run ESLint for all files in the `pages/` directory. However, you can specify other custom directories to run by using the `--dir` flag in `next lint`:
+By default, Next.js will run ESLint for all files in the `pages/`, `components/`, and `lib/` directories. However, you can specify other custom directories to run by using the `--dir` flag in `next lint`:
```

Comment:
Is there a way to change these defaults? E.g. maybe I don't want `lib`. Could be follow up later 😄

Reply:
Just added the option to configure this via `next.config.js`:

```js
module.exports = {
  eslint: {
    dirs: ['pages', 'components'] // Only run ESLint on pages and components directories during builds
  }
}
```

https://github.com/vercel/next.js/commit/f57a0469e7f6799d6d3ac0db79a783a7078b0304#diff-ea22ff5b8d78b521a09affbcc239809dc95143cda0370d9992420ab86172ea58